src/tools/compiletest/src/header.rs

```diff
@@ -676,14 +676,17 @@ pub fn line_directive<'line>(
 /// names. This is **not** an exhaustive list of all possible directives. Instead, this is a
 /// best-effort approximation for diagnostics.
```

Comment:
Is it still the case that this is not an exhaustive list? If not can you remove the respective part of this comment.